modules/core/src/test/java/org/apache/ignite/spi/discovery/tcp/TcpDiscoveryNetworkIssuesTest.java

```diff
@@ -153,6 +158,11 @@ public class TcpDiscoveryNetworkIssuesTest extends GridCommonAbstractTest {
 
         cfg.setLocalHost(localhost);
 
+        if (testLog != null) {
+            cfg.setGridLogger(testLog);
+            testLog = null;
```

Comment:
Let's reset `testLog` somewhere else. This function should not make this decision, only follow the command to set the log.